source/opt/debug_info_manager.cpp

```diff
@@ -849,7 +849,7 @@ void DebugInfoManager::ClearDebugInfo(Instruction* instr) {
     fn_id_to_dbg_fn_.erase(fn_id);
   }
   if (instr->GetShader100DebugOpcode() ==
-      NonSemanticShaderDebugInfo100DebugFunction) {
+      NonSemanticShaderDebugInfo100DebugFunctionDefinition) {
```

Comment:
Not very familiar with that yes, but if I understand:
 - we were looking for DebugFunction but querying the operand index from DebugFunctionDefinition.
 Why change the type we search and not fix the operand index?
 It looks like DebugFunction can be used to describe both a declaration and a definition, so shouldn't we clear both?

Reply:
This was a typo from a previous commit. DebugFunctionDefinition was intended all along. This just fixed the bug.

DebugFunction is covered by the preceding if statement since it is in both the OpenCL and Shader specs.